web/themes/custom/pineapple/sass/base/forms/_filters-form.scss

```diff
@@ -50,6 +50,10 @@
       }
     }
 
+    .form-type-checkbox {
```

Comment:
Use a different class that's more specific to the field

Reply:
Do you know if I can add a class to this in the drupal?

Reply:
I used classes like this in the same sass file.... eg.- form-type-select. Because they are inside the views exposed form, I don't think this is an issue?

Reply:
You can but you don't need to, the field already has a class that references the field name

Reply:
Fairs, done.